Tests/FeedStoreIntegrationTests.swift

```diff
@@ -3,7 +3,7 @@
 //
 
 import XCTest
-import FeedStoreChallenge
+@testable import FeedStoreChallenge
```

Comment:
`@testable` is not needed in this file - you can remove it.

```suggestion
import FeedStoreChallenge
```